setup_headless_testing.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+echo "This setup script was written for Ubuntu 16.04.1 and may not work elsewhere."
+echo "Updating system ..."
+sudo apt-get update -qq
+echo "Adding requisite QtWebkit libs ..."
+sudo apt-get install libqt4-dev libqtwebkit-dev -y -qq
```

Comment:
Capybara-webkit will throw a warning saying you should switch to 5.0+ for future builds, but that doesn't compile yet, so there's no advantage to switching yet.